### PR TITLE
debian: Bump eos-parental-controls-0 dependency for new symbols

### PR DESCRIPTION
**Pull request by pwithnall**
We need epc_app_filter_is_user_installation_allowed().

Signed-off-by: Philip Withnall <withnall@endlessm.com>

https://phabricator.endlessm.com/T24457

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -28,7 +28,7 @@ Build-Depends:
  libcap-dev,
  libdw-dev,
  libelf-dev,
- libeos-parental-controls-0-dev,
+ libeos-parental-controls-0-dev (>= 0+dev53.de16300bem1),
```
Comment (wjt) [inline debian/control]: Why is this needed? Won't the newest version always be included in the Jenkins and OBS build environments, and in the image?

Comment (pwithnall) [reply debian/control]: Thinking about the recent runtime failures from various people who’ve built the Shell locally and haven’t updated their build dependencies, I thought it might be wise to try and avoid that here too. If you strongly feel it isn’t necessary then I’m happy to drop the PR>

Comment (wjt) [reply debian/control]: I suppose it can't hurt!
